gapic/src/main/com/google/gapid/models/Memory.java

```diff
@@ -549,8 +549,12 @@ public String getTypeFormatted() {
 
     public String getValueFormatted() {
       if (type.getTyCase() == TyCase.ENUM && type.getEnum().hasConstants()) {
-        return ConstantSets.find(typesModel.constants.getConstants(type.getEnum().getConstants()),
-            value.getPod()).getName();
+        Service.ConstantSet constantSet = typesModel.constants.getConstants(type.getEnum().getConstants());
+        if (constantSet != null) {
```

Comment:
if `constantSet` is null here, it has not been loaded and you should schedule the load and ensure the UI refreshes once it has been loaded.